diesel/src/macros/mod.rs

```diff
@@ -928,13 +928,13 @@ macro_rules! __diesel_table_query_source_impl {
 ///
 /// * `child_table` is the Table with the Foreign key.
 ///
-/// So given the Table decaration from [Associations docs](http://docs.diesel.rs/diesel/associations/index.html)
+/// So given the Table decaration from [Associations docs](https://docs.diesel.rs/diesel/associations/index.html)
 ///
 /// * The parent table would be `User`
 /// * The child table would be `Post`
 /// * and the Foreign key would be `Post.user_id`
 ///
-/// For joins that do not explicitly use on clauses via [`JoinOnDsl`](http://docs.diesel.rs/diesel/prelude/trait.JoinOnDsl.html)
+/// For joins that do not explicitly use on clauses via [`JoinOnDsl`](https://docs.diesel.rs/diesel/prelude/trait.JoinOnDsl.html)
```

Comment:
```suggestion
/// For joins that do not explicitly use on clauses via [`JoinOnDsl`](prelude/trait.JoinOnDsl.html)
```

Reply:
For this, does simply ``[`JoinOnDsl`]`` or ``[`JoinOnDsl`](crate::prelude::JoinOnDsl)`` work? I remember using crate relative links and they worked preety nicely.

Reply:
I think it's okay as it is.